executor/write.go

```diff
@@ -604,7 +604,7 @@ type InsertExec struct {
 
 	OnDuplicate []*expression.Assignment
 
-	Priority int
+	Priority ast.PriorityEnum
```

Comment:
This field seems to be useless according to https://github.com/pingcap/tidb/pull/3523